setup.py

```diff
@@ -42,6 +42,8 @@
             "waitress>=2.0.0",
             "Flask<2",
             "Flask>=1.0.2",
+            # Due to https://github.com/pallets/jinja/issues/1585
+            "markupsafe<2.1",
```

Comment:
We're using `Flask<2` to have a common dependency with microcosm-flask, otherwise the installed version will depend on install order. That means we can't use the latest version of Jinja, so we have to cap the version here.